internal/service/advancedcluster/move_state_test.go

```diff
@@ -0,0 +1,79 @@
+package advancedcluster_test
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+func TestAccAdvancedCluster_move_not_supported_sdkv2(t *testing.T) {
```

Comment:
```suggestion
func TestAccAdvancedCluster_moveNotSupportedSdkv2(t *testing.T) {
```
nit to keep consistency on naming of tests

Reply:
thanks, changed